pkg/controller/configmap/config.go

```diff
@@ -316,6 +297,102 @@ func validateConfig(cfg *ini.File, sec string, key string) error {
 	return nil
 }
 
+// Validate node agent specific options
+func validateNodeAgentOptions(configmap *corev1.ConfigMap, cfg *ini.File) []error {
+	errs := []error{}
+	var err error
+	// Check MTU value
+	mtu := cfg.Section("nsx_node_agent").Key("mtu").Value()
+	_, err = strconv.Atoi(mtu)
+	if err != nil {
+		appendErrorIfNotNil(&errs, errors.Wrapf(err, "mtu is invalid"))
+	}
+	return errs
+}
+
+// Validate common options for both ncp and node agent
+func validateCommonOptions(configmap *corev1.ConfigMap, cfg *ini.File) []error {
+	errs := []error{}
+	data := &configmap.Data
+	var err error
+	// Check DEFAULT section log_file option
+	if cfg.Section("DEFAULT").Key("log_file").Value() != "" {
```

Comment:
Could you help explain why log_file cannot be changed

Reply:
This is a workaround solution. Currently, ncp configmap and node agent configmap will use the same option for log_file. If user don't set log_file, the ncp and node agent will default log file name. The log file are different for ncp and node agent log. Once user set log_file option, ncp and node 
the log_file will be used for bootstrap, ncp and node agent pods. All pods will write log in the same log_file. However, ncp and node agent write log_file with ncp user, bootstrap pod
write log_file with root user. when bootstrap pod create log_file before than ncp/node-agent pods is trying to create it.  The log_file file permission will be set
-rw-r--r--.  1 root root     9186 Dec 13 03:11 test_logfile
It means other uses other than root can not write this file any more. So, In such case, the ncp/node-agent will hit Permission denied issue when writing log_file.